lib/ci-stack.ts

```diff
@@ -33,19 +33,31 @@ export class CIStack extends Stack {
       },
     });
 
-    const useSslParameter = new CfnParameter(this, 'useSsl', {
-      description: 'If the jenkins instance should be access via SSL',
-      allowedValues: ['true', 'false'],
+    const useSslParameter = this.node.tryGetContext('useSsl');
+    if (useSslParameter !== 'true' && useSslParameter !== 'false') {
+      throw new Error('useSsl parameter is required to be set as "true" or "false"');
```

Comment:
Still see the quotes :) Please remove it. Errors are important and need to be clear